tutorials/db-migration-spanner-handle-increasing-pks/index.md

```diff
@@ -0,0 +1,363 @@
+        
+        IMPORT STATIC CustomFunctions.*;
+        
+        CREATE APPLICATION mysql_to_spanner_initial_load;
+        
+        CREATE FLOW sample1;
+        
+        CREATE OR REPLACE SOURCE mysql_dbreader USING Global.DatabaseReader (
+          Username: 'root',
+          DatabaseProviderType: 'Default',
+          FetchSize: 10,
+          adapterName: 'DatabaseReader',
+          QuiesceOnILCompletion: true,
+          Password_encrypted: 'false',
+          ConnectionURL: 'jdbc:mysql://PRIMARY_IP:3306/employeedb',
+          Tables: 'employeedb.signin_log',
+          Password: 'password123',
+          ReturnDateTimeAs: 'String')
+        OUTPUT TO sample_raw;
+        
+        CREATE STREAM SampleModifiedData OF Global.WAEvent;
+        
+        CREATE CQ POPULATE_SAMPLE_TABLE
+        INSERT INTO SampleModifiedData
+        SELECT * from sample_raw
+        MODIFY(data[0] = bitReverseInt(data[0]));
+        
+        CREATE TARGET spanner_sample USING Global.SpannerWriter (
+          InstanceID: 'spanner-tgt',
+          ServiceAccountKey: '/opt/striim/striim-spanner-key.json',
+          BatchPolicy: 'EventCount: 100, Interval: 10s',
+          ParallelThreads: '',
+          Tables: 'employeedb.signin_log,employeedb.signin_log' )
+        INPUT FROM SampleModifiedData;
+        
+        END FLOW sample1;
+        
+        END APPLICATION mysql_to_spanner_initial_load;
+
+2. Login to striim web ui. If needed, visit [deployment manager page](https://console.cloud.google.com/dm/deployments)
+ access striim web ui link and credentials. 
+3. Click on Apps > + Add App (on top right).
+4. Click import existing app and choose the mysql_to_spanner_initial_load.tql created in step 1. And click import.
+5. In the workflow, replace Primary_IP with MySQL’s ip address. Click mysql_dbreader component and change the Primary_IP 
+in the right window showing connection url (screenshot below). And click save
+![initial load](4_striim_initial_load.png)
+
+6. Created > Deploy App > Deploy
+![initial load deploy](5_striim_deploy.png)
+
+7. Deploy > Start App. 
+8. You will soon observe rows flowing through striim and data being inserted inside the cloud spanner.
+9. On spanner gui verify that rows have been written with bit reversed id values.
+![initial complete](6_spanner_il_complete.png)
+
+### Create continuous replication (CDC) pipeline
+Once initial load is complete, then using CDC pipeline you can keep replicating the new changes into Spanner.  
+Keeping MySQL and Spanner in sync. It reads data from mysql's binary logs using Striim's MysqlReader component.  
+
+In production you would use `StartTimestamp` property to specify binary log position (i.e. time at which initial load started).  
+Also, you might like to create a `CheckpointTable` so that [striim can recover](https://www.striim.com/docs/en/spanner-writer.html) from a crash.
+However these concepts are out of scope for this tutorial.
```

Comment:
```suggestion
In production you would use `StartTimestamp` property to specify binary log position (i.e. timestamp at which initial load was started) from which the CDC application should start replicating.  
You should also create a [Checkpoint table](https://www.striim.com/docs/en/spanner-writer.html) so that Striim can recover from a failure.
However these concepts are out of scope for this tutorial.
```